modules/ROOT/pages/storage.adoc

```diff
@@ -128,3 +128,41 @@ storage:
 ----
 
 NOTE: You don't need the `path` or `with_mount_unit` keys; FCOS knows that the root partition is special and will figure out how to find it and mount it.
+
+== Encrypted storage (LUKS)
+
+Here is an example to configure a LUKS device at `/var/lib/data`.
+
+[source.yaml]
+----
+variant: fcos
+version: 1.2.0-experimental
+storage:
+  luks:
+    - name: data
+      device: /dev/vdb
+  filesystems:
+    - path: /var/lib/data
+      device: /dev/disk/by-id/dm-name-data
+      format: xfs
+      label: DATA
+      with_mount_unit: true
+----
+
+.Moving the root filesystem to LUKS
+[source.yaml]
+----
+variant: fcos
+version: 1.2.0-experimental
+storage:
+  luks:
+    - name: root
+      device: /dev/vdb
```

Comment:
I think it will be more common to reprovision LUKS in place. So this would be `/dev/disk/by-partlabel/root` (see the added test in https://github.com/coreos/fedora-coreos-config/pull/609).

Reply:
So then we'd need `wipeVolume: true` here too like in https://github.com/coreos/fedora-coreos-config/pull/616.

Reply:
Sure, swapped it over to doing an in place override.